desktop_version/src/Render.cpp

```diff
@@ -1381,6 +1381,30 @@ void gamerender()
         {
             graphics.drawtowerspikes();
         }
+
+        // Editor ghosts!
+        if (game.ghostsenabled)
+        {
+            if (map.custommode && !map.custommodeforreal)
+            {
+                if (game.gametimer % 3 == 0)
+                {
+                    int i = obj.getplayer();
+                    GhostInfo ghost;
+                    ghost.rx = game.roomx-100;
+                    ghost.ry = game.roomy-100;
+                    ghost.x = obj.entities[i].xp;
+                    ghost.y = obj.entities[i].yp;
+                    ghost.col = obj.entities[i].colour;
+                    ghost.frame = obj.entities[i].drawframe;
+                    ed.ghosts.push_back(ghost);
+                }
+                if (ed.ghosts.size() > 100)
+                {
+                    ed.ghosts.erase(ed.ghosts.begin());
+                }
+            }
+        }
```

Comment:
There should probably be an `#if !defined(NO_CUSTOM_LEVELS)` guard here, otherwise a `-DCUSTOM_LEVEL_SUPPORT=DISABLED` build fails.